automated-testing/performance-testing/readme.md

```diff
@@ -1,67 +1,357 @@
+    enhancements introduced due to a change
+
+- To **support system tuning**
+
+  - Comparing performance characteristics of a system for different system
+    configurations
+
+- To assist **capacity planning**
 
-* To help in assessing whether a **system is ready for Release**:
-  * Estimating / Predicting the performance characteristics (such as response time, throughput) which an application is likely to have when it is released in to production. The results can help in predicting the satisfaction level of the users when interacting with the system. The predicted values can also be compared with agreed values (success criteria) for the performance characteristics when available.
-  * To help in accessing the adequacy of the infrastructure / managed service SKUs to meet the desired performance characteristics of a system
-  * Identifying bottlenecks and issues with the application at different load levels
-* To compare the **performance impact of application changes**
-  * Comparing the performance characteristics of an application after a change to the values of performance characteristics during previous runs (or baseline values), can provide an indication of performance issues or enhancements introduced due to a change
-* To **support system tuning**
-  * Comparing performance characteristics of a system for different system configurations
+  - Capacity planning is the process of determining what type of hardware and
+    software resources are required to run an application with the given user
+    load. Poor performing applications require more hardware and thus are more
+    expensive to operate. Capacity planning involves identifying business
+    expectations, the periodic fluctuations of application usage, considering
+    the cost of running the hardware and software infrastructure.
+
+- To reduce **downtime**
+
+  - Systems that go down cost money. For every second thought. Amazon Web
+    services is down, is a loss of eleven hundred dollars per second. And for
+    financial services firms’ downtime could mean lawsuits and customer
+    attrition.
 
 ## Key Performance Testing categories
 
-<!-- markdownlint-disable no-duplicate-heading -->
-### Performance Testing
-<!-- markdownlint-enable no-duplicate-heading -->
+Performance testing is a broad topic. There are many areas where you can perform
+tests. In broad strokes you can perform tests on the backend and on the front
+end. You can test the performance of individual components as well as testing
+the end\-to\-end functionality.
+
+There are several categories of tests as well:
 
-  This category is the super set of all sub categories of performance related testing. It validates/determines the speed, scalability or reliability characteristics of the system under test. Performance testing focuses on achieving the response times, throughput, and resource utilization levels which meet the performance objectives of a system
+### Load Testing
```

Comment:
please re-add the link to the load-testing document

Reply:
Yes, agreed. Updated to avoid names.

Reply:
The title used to link to "./load-testing.md" that focuses on load-testing. Can you re-add the link, either to the title or somewhere in the paragraph.